net-wireless/nitroshare/nitroshare-0.4.0~git.recipe

```diff
@@ -60,15 +67,20 @@ defineDebugInfoPackage nitroshare$secondaryArchSuffix \
 
 BUILD()
 {
-	cmake -DCMAKE_INSTALL_PREFIX:PATH=$appsDir .
+	cmake -DCMAKE_BUILD_TYPE=Release \
+	$cmakeDirArgs \
```

Comment:
Doesn't use GnuInstallDirs it seems, but you can use variables included like you did for $includeDir

https://github.com/nitroshare/nitroshare-desktop/blob/master/CMakeLists.txt#L39